### PR TITLE
DAT-4566  Chocolatey distribution - openjdk8 to openjdk17

### PR DESCRIPTION
**Pull request by jandroav**

### CODE REVIEW

File: templates/liquibase.nuspec.TEMPLATE [hunk 1]
```diff
@@ -22,7 +22,7 @@ Liquibase is an open source, database-independent library for tracking, managing
     <projectSourceUrl>https://github.com/liquibase/liquibase</projectSourceUrl>
     <mailingListUrl>http://www.liquibase.org/blog/</mailingListUrl>
     <dependencies>
-      <dependency id="openjdk8"/>
+      <dependency id="openjdk" version="17.0.2"/>
```
Comment (jnewton03) [inline templates/liquibase.nuspec.TEMPLATE:25]: can we update this dep to temurin 17 since openjdk is deprecated?

Comment (jandroav) [reply templates/liquibase.nuspec.TEMPLATE]: done @jnewton03

**Comment by jandroav** [comment]
do we need to stick to a specific version? https://community.chocolatey.org/packages/Temurin17#versionhistory

File: .github/workflows/deploy-package.yml [hunk 2]
```diff
@@ -21,31 +21,31 @@ jobs:
...
+
+      - name: Build nupkg Chocolatey package
+        run: ant make-nuspec
+
+      - name: Choco pack
+        run: |
+          choco pack
+
+      - name: Install Temurin 17
+        run: choco install temurin17
```
Comment (jnewton03) [inline .github/workflows/deploy-package.yml:42]: @jandroav @filipelautert are we ok with the latest version of temurin 17?  Looks like it is `17.0.10`

Comment (jandroav) [reply .github/workflows/deploy-package.yml:42]: I would go with `temurin17` to always have the latest fixes.

Comment (jnewton03) [reply .github/workflows/deploy-package.yml:42]: I know we had an issue before with a particular minor version:
https://github.com/liquibase/liquibase/commit/bb5d00e8a24f795b5cdd85fe1ef322cd8c3d2e54

@filipelautert is this still an issue in the latest temurin?

**Review by jnewton03 — APPROVED** [review]

**Pull request merged**